modules/machineconfig-modify-registry.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * installing/post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machineconfig-modify-registry_{context}"]
+= Configuring container image registry settings
+
+Settings that define the registries {product-title} uses to get container images
+are held in the `/etc/containers/registries.conf` by default. In that file,
+you can set registries to not require authentication (insecure), point to
+mirrored registries, or set which registries are searched for unqualified
+container image requests.
+
+Rather than change `registries.conf` directly, you can drop configuration files
+into the `/etc/containers/registries.d` directory that are then automatically
+appended to the system's existing `registries.conf` settings.
+
+This procedure describes how to create a `registries.d` file
+(`/etc/containers/registries.s/99-worker-unqualified-search-registries.conf`)
+that adds `quay.io` as an unqualified search registry (one that {product-title}
+can search when it tries to pull an image name does not include the registry name).
```

Comment:
s/name does not include/name that does not include/

Reply:
Done.